web/src/scenes/actions/hooks/use-fetch-actions.tsx

```diff
@@ -0,0 +1,21 @@
+import { useAppStore } from "src/stores/appStore";
```

Comment:
We agreed that we are implementing automatic type generation. It will take time to update the entire app. The row you point to in this comment uses the zustand store, from an earlier implementation. 
Here we have two options here:
1. We are gradually introducing new functionality (codegen, react-hook-forms)
2. We allocate time for a complete refactor of the application.